admin/app/com/lucidchart/piezo/admin/controllers/JobDataHelper.scala

```diff
@@ -34,7 +34,7 @@ trait JobDataHelper {
 
   implicit def jobDataMap: Mapping[Option[JobDataMap]] =
     optional(
-      list(mapping("key" -> text, "value" -> text)(DataMap.apply)(DataMap.unapply))
+      list(mapping("key" -> text, "value" -> text)(DataMap.apply)(r => Some((r.key, r.value))))
```

Comment:
Super-nit: Whether you accept this suggestion is completely up to you, but I strongly dislike single-letter variable names unless they're referring to an index. Could you make this more descriptive?